ide/csl.api/src/org/netbeans/modules/csl/editor/overridden/Bundle.properties

```diff
@@ -19,7 +19,7 @@ CAP_Implements=Implements
 CAP_Overrides=Overrides
 #{0}: implements/overrides what
 TP_Implements=Implements method from : {0}
-TP_Overrides=Overrides method from: {0}
+TP_Overrides=Overrides from: {0}
```

Comment:
@sdedic Is this change OK? Constants and fields are overridden in PHP.

Reply:
I guess so ?

Reply:
Thanks!